setup/features/server-group-launch-settings/aws-ec2/index.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: single
+title:  "Overview"
+sidebar:
+  nav: setup
+---
+
+## Overview
+An AWS EC2 Server Group offers a number of launch setting configurations that can enhance your experience.
+
+### Launch Settings
+An AWS Server Group can be set up with two types of launch settings. However, <b>you must enable Launch Templates support for all of your applications if you want to use the latest AWS features.
+AWS strongly recommends using Launch Templates over Launch Configurations because Launch Configurations do NOT provide full functionality for Amazon EC2 Auto Scaling or Amazon EC2.</b>
```

Comment:
"launch templates" and "launch configurations" are written all lowercase; we don't capitalize them in the EC2 docs. I can see that you might want them to stand out here. You could italicize them here to make them stand out.